### PR TITLE
SDK-458: Update WP landing page summary and banner

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

**Review by echarrod — APPROVED** [review]

File: yoti/readme.txt [hunk 1]
```diff
@@ -5,109 +5,56 @@ Tags: identity, verification, login, form, 2 factor, 2 step authentication, 2FA,
...
-This will then lead you to create a new application.
-
-Specify the basic details of your application such as the name, description and optional logo. These details can be whatever you desire and will not affect the plugin's functionality.
-
-The Data tab - Specify any attributes you'd like users to share. You must select at least one. If you plan to allow new user registrations, we recommended choosing Given Name(s), Family Name and Email Address at a minimum.
-
-The Integration tab - Here is where you specify the callback URL. The URL can be found by going to your Yoti settings page which is located in your WordPress admin dashboard.
-
-NOTE: If you get redirected to your WordPress frontpage instead of the Admin area, simply add /wp-admin to the URL.
+Visit the Yoti Dashboard [here](https://www.yoti.com/dashboard/login) to create a new application for your organisation/business.
```
Comment (Andrieg) [inline yoti/readme.txt]: Let's update this URL to https://www.yoti.com/dashboard/login-organisations

File: yoti/readme.txt [hunk 2]
```diff
@@ -5,109 +5,56 @@ Tags: identity, verification, login, form, 2 factor, 2 step authentication, 2FA,
...
 
-The Key tab - Here is where your keys are generated which will be inputted into the plugin settings. You will need to download your pem file and store it somewhere safe as it will be used as part of the plugin set up.
+Specify the basic details of your application such as the name, description and optional logo. These details can be whatever you desire and will not affect the plugin’s functionality.
 
-= Step 3: Plugin Setup =
+The main page - Edit the application and set your website URL in the 'Application domain' section, e.g https:yourwebsite.com
 
-To set things up, navigate on WordPress to Settings > Yoti. You will be asked to add the following information:
+The Scenarios tab - Edit the default scenario and fill in the following sections:
```
Comment (Andrieg) [inline yoti/readme.txt]: The Scenario tab - Scenarios are different instances where you request users for information using Yoti. e.g verify your users' age online or in person, or quickly sign in users to your website without passwords.  Scenarios lets you have different instances of the Yoti plugin widget on your site.

-Specify a name for your scenario.
-Specify what information you want to request with this scenario.
-And finally, provide a callback URL so we know where to send your users after they have used Yoti. This URL must be a subdomain of your applications' domain.

Comment (sidibos) [reply yoti/readme.txt]: 👍

**Comment by sidibos** [comment]
@KiranBali , @Andrieg do you guys want to have a final look before I release it?

**Review by Andrieg — APPROVED** [review]

File: yoti/readme.txt (hunk 1 above)
Comment (KiranBali) [reply yoti/readme.txt]: agreed

File: yoti/readme.txt [hunk 3]
```diff
@@ -5,109 +5,56 @@ Tags: identity, verification, login, form, 2 factor, 2 step authentication, 2FA,
...
-* Automatic login to create blogs with ease
-* No need for passwords
-* Remember me functionality
-* The ability for user to link and unlink WordPress account to Yoti account
-
-== Installation ==
+== Why does your website need Yoti ==
 
-We have a few simple steps we need you to fill before you get started:
+* Stops keyloggers. By not having to type usernames and passwords.
```
Comment (KiranBali) [inline yoti/readme.txt]: Can we have a comma instead?

File: yoti/readme.txt (hunk 2 above)
Comment (KiranBali) [reply yoti/readme.txt]: can we add around how we only support ONE scenario

**Review by KiranBali — CHANGES_REQUESTED** [review]

File: yoti/readme.txt (hunk 2 above)
Comment (Andrieg) [reply yoti/readme.txt]: I didn't know the plugin can only support one scenario. You can completely ignore my comment on the Scenario tab and go with what you had before.

**Comment by sidibos** [comment]
@KiranBali, @Andrieg I have updated the Readme, could you guys take a look again.

**Review by KiranBali — APPROVED** [review]

**Pull request merged**